.github/actions/generate-matrix/action.yml

```diff
@@ -39,6 +39,10 @@ inputs:
   default_tests:
     description: "Default JUnit5 tests that will be executed"
     default: ""
+  runnerArch:
+    description: "Architecture of GitHub runner"
+    required: false
+    default: "amd64"
```

Comment:
Just out of curiosity - why it's `amd64` when it was `arm64` before?

Reply:
Yeah it was just question because I'm curious. I'm fine with leaving it as it is.